content/en/security/application_security/threats/setup_and_configure.md

```diff
@@ -231,6 +231,46 @@ Support for query strings is not available for Flask.
 
 {{< /programming-lang >}}
 
+{{< programming-lang lang="serverless" >}}
+
+<div class="alert alert-info">ASM support for AWS Lambda is in beta. Threat detection is done by using the Lambda extension.<br><br> Don’t see your desired technology listed here? Datadog is continually adding additional support. Fill out <a href="https://forms.gle/gHrxGQMEnAobukfn7">this short form to send us details</a>.</div>
+
+
+### Supported cloud environments
+
+- AWS Lambda (beta)
+
+### Supported languages and their requirements
+
+Node version x.x
+: If you are bundling using webpack or esbuild, [mark the Datadog libraries as external][4].
+
+Python version x.x
+: If your Python Lambda functions are written in Python 3.6 or less, or you previously set up your Lambda functions using the Datadog Forwarder, read [instrumenting using the Datadog Forwarder][2].
+
+Java version x.x
+: To fully instrument your serverless application with distributed tracing, your Java Lambda functions must use the Java 8 Corretto (`java8.al2`) or Java 11 (`java11`) runtimes with at least 1024MB of memory.
+: If you use the Datadog Lambda layers `dd-trace-java:4` (or older) and `Datadog-Extension:24` (or older), follow the instructions in [Upgrade Instrumentation for Java Lambda Functions][3].
+
+Go version x.x
+: If your Go Lambda functions use runtime `go1.x` and you cannot migrate to the `provided.al2` runtime, you must [instrument using the Datadog Forwarder][2] instead.
+
+.NET version x.x
```

Comment:
Remove x.x version number since it's dependent on the extension version, not language

Reply:
Universal instrumentation API requires a min Java tracer version.
Adding a suggestion in a few minutes to add it.